numpy/core/src/npymath/npy_math_internal.h.src

```diff
@@ -887,7 +887,7 @@ npy_popcountu@c@(npy_@type@ a)
 /* use built-in popcount if present, else use our implementation */
 #if (defined(__clang__) || defined(__GNUC__)) && NPY_BITSOF_@STYPE@ >= 32
     return __builtin_popcount@c@(a);
-#elif defined(_MSC_VER) && NPY_BITSOF_@STYPE@ >= 16
+#elif defined(_MSC_VER) && NPY_BITSOF_@STYPE@ >= 16 && !defined(_M_ARM64)
```

Comment:
Thanks for the fix! I'm not aware of ARM(hence I missed it), but should we add `_M_ARM` as well?

Reply:
Done!